CMakeLists.txt

```diff
@@ -105,3 +105,19 @@ install(FILES
   "${PROJECT_BINARY_DIR}/poco_vendorConfig.cmake"
   "${PROJECT_BINARY_DIR}/poco_vendorConfig-version.cmake"
   DESTINATION share/${PROJECT_NAME}/cmake)
+
+find_package(ament_cmake_test QUIET)
+if(BUILD_TESTING)
+  find_package(ament_cmake_copyright QUIET)
+  find_package(ament_cmake_lint_cmake QUIET)
+  find_package(ament_cmake_xmllint QUIET)
```

Comment:
Why pick a few specific linters instead of running them all?

Reply:
For vendored packages, those are the linters used. Didn't include the complete ament_lint_common suite, because this requires the ament_cmake environment workflow (calling ament_package), and this package is cmake based.